agriculture_shop/static/src/js/my_tour.js

```diff
@@ -0,0 +1,23 @@
+/** @odoo-module **/
+
+import { _t } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+import { markup } from "@odoo/owl";
+
+registry.category("web_tour.tours").add("knowledge_tour", {
+  url: "/web",
+  rainbowManMessage: _t("Congrats :)"),
```

Comment:
Either you add a step to show it, either you remove this line (I'd go for option 2)
```suggestion
```